### DISCUSSION TITLE
Stop hotspot traffic when VPN is down "feature"

### DISCUSSION THREAD

**Discussion by Back-Web** [Mygod/VPNHotspot#601] [General]
Can anyone tell me if there is a possibility of blocking the connection of "shared" hotspot users if the VPN connection drops, that is, blocking all access that is not through the VPN connection?
I know it is possible to do this by implementing a firewall rule on Android by interface, but I would like to know if any version of the APP already does this.

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/issues/47#issuecomment-448463139

**Reply by Back-Web** [reply]
Tks, worked!!! The fallback upstream interface set to none and disable IPv6 tethering.
